CRM/Contact/Form/CustomData.php

```diff
@@ -185,11 +185,11 @@ public function buildQuickForm() {
         );
         if ($isMultiple) {
           $this->assign('multiRecordDisplay', $this->_multiRecordDisplay);
-          $saveButtonName = $this->_copyValueId ? 'Save a Copy' : 'Save';
+          $saveButtonName = $this->_copyValueId ? ts('Save a Copy') : ts('Save');
           $this->addButtons(array(
               array(
                 'type' => 'upload',
-                'name' => ts('%1', array(1 => $saveButtonName)),
```

Comment:
Not sure if you'll be surprised to learn grep turns up 8 other instances of `ts('%1'`